ts/webui/src/static/style/common/common.scss

```diff
@@ -0,0 +1,22 @@
+.ellipsis {
```

Comment:
The path looks a little bit strange. Why it's not `style/common.scss`?

Reply:
thanks~ it's easy to refine.